Libplanet.Net/Transports/TcpTransport.cs

```diff
@@ -412,10 +402,12 @@ public void BroadcastMessage(IEnumerable<BoundPeer> peers, Message message)
                 throw new ObjectDisposedException(nameof(TcpTransport));
             }
 
-            foreach (var peer in peers)
-            {
-                _ = SendMessageAsync(peer, message, _runtimeCancellationTokenSource.Token);
-            }
+            peers.AsParallel().ForAll(
+                peer => Task.Run(() => SendMessageAsync(
+                    peer,
+                    message,
+                    TimeSpan.FromSeconds(1),
+                    _runtimeCancellationTokenSource.Token)));
```

Comment:
Do we need to wrap by `Task.Run()` here?

```suggestion
            peers.AsParallel().ForAll(
                peer => SendMessageAsync(
                    peer,
                    message,
                    TimeSpan.FromSeconds(1),
                    _runtimeCancellationTokenSource.Token));
```

Reply:
I couldn't figure out a better way to do it. Without `Task.Run()`, we get a [CS4014] compiler warning. If we use `async` and `await` inside `ForAll()`, we get an error. 🙁

[CS4014]: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs4014

Reply:
Well, got it. `.SendMessageAsync()` was marked as `async` keyword so compiler seemed complain. (Sometimes I get confused between API that just returns a `Task` (e.g. `Task.Delay()`) and marked as `async` 😕).

I think it seems knitpick, but we can write as below.

```suggestion
            peers.AsParallel().ForAll(
                peer => async () => await SendMessageAsync(
                    peer,
                    message,
                    TimeSpan.FromSeconds(1),
                    _runtimeCancellationTokenSource.Token));
```

Reply:
🆗